pyroomacoustics/doa/srp.py

```diff
@@ -63,24 +83,54 @@ def _process(self, X):
 
         CC = []
         for k in self.freq_bins:
-            CC.append( np.dot(pX[:,k,:], np.conj(pX[:,k,:]).T) )
+            CC.append(np.dot(pX[:, k, :], np.conj(pX[:, k, :]).T))
         CC = np.array(CC)
 
+        M = self.L.shape[1]
+        ar = np.arange(M)
+        av = ar[:, None]
+
+        # the two masks here allow to select the upper triangular part
```

Comment:
sorry for nitpicking, there's only one mask left now

Reply:
😆 fixed